about.html

```diff
@@ -9,7 +9,34 @@
+        </li> --><li class=""><a class="nav-link" href="cases" data-i18n="nav-cases">應用案例</a></li><li class=""><a class="nav-link" href="about" data-i18n="nav-about">關於我們</a></li><li class=""><div class="dropdown lang-selector-dropdown"><button class="lang-selector dropdown-toggle" data-toggle="dropdown" aria-haspopup="true" aria-expanded="false">中文</button><ul class="dropdown-menu dropdown-menu-right"><li><button data-locale="zh" class="switch-locale dropdown-item">中文</button></li><li><button data-locale="en" class="switch-locale dropdown-item">English</button></li><li><button data-locale="jp" class="switch-locale dropdown-item">日本語</button></li></ul></div></li></ul></div></nav></header><section id="section-banner"><!-- <canvas id="banner-canvas">Canvas not supported.</canvas> --><picture><source srcset="assets/about/about-banner@2x.webp" type="image/webp"><source srcset="assets/about/about-banner.png, assets/about/about-banner@2x.png 2x"><img src="assets/about/about-banner.png" class="img-fluid" alt="banner"></picture></section><section id="section-slogon"><div class="container thin-container"><div class="row align-items-center"><div class="col-8 col-lg-3 slogon-img"><picture><source srcset="assets/about/logo-white@2x.webp" type="image/webp"><source srcset="assets/about/logo-white.png, assets/about/logo-white@2x.png 2x"><img src="assets/about/logo-white.png" class="img-fluid" alt="BiiLabs"></picture></div><div class="col-12 col-lg-9 slogon-text" data-i18n="about-slogon">BiiLabs致力於開發分佈式分類帳技術，以克服萬物互聯時代的挑戰，包括智能移動性和能源管理等。 其核心技術可幫助各個行業進行數字化轉型，並為客戶解決關鍵問題，例如信任、安全性、增長和效率。 BiiLabs的目標是成為萬物互連世界的領先技術提供商。</div></div></div></section><section id="section-milestone"><div class="container thin-container"><h2 class="section-title about-milestone" data-i18n="about-milestone">里程碑</h2><br><br></div><div class="container d-none d-md-block"><!-- <object type="image/svg+xml" data="assets/about/milestone_line_static.svg">
+      </object> --><style type="text/css">.st0,
```

Comment:
Would you use HTML Tidy or similar software to indent HTML source?

Reply:
fixed